lib/util.js

```diff
@@ -44,6 +44,10 @@ exports.getLabels = function (labelNames, args) {
 	return acc;
 };
 
+exports.wait = async function (ms) {
```

Comment:
I don't think we need to expose this out, it's enough to use it in the single test